WeakAuras/GenericTrigger.lua

```diff
@@ -829,9 +829,21 @@ function GenericTrigger.Add(data, region)
             else
               error("Improper arguments to WeakAuras.Add - no event prototype can be found for event type \""..trigger.event.."\" and default prototype reset failed.");
             end
-          elseif(trigger.event == "Combat Log" and not (trigger.subeventPrefix..trigger.subeventSuffix)) then
-            error("Improper arguments to WeakAuras.Add - event type is \"Combat Log\" but subevent is not defined");
           else
+            if (trigger.event == "Combat Log") then
```

Comment:
This used to be done in special set methods for those two properties. Moving the code here, makes the code around the trigger options not have to care about that special case of the combat log trigger.